docs/source/index.rst

```diff
@@ -154,6 +154,7 @@ Welcome to AiiDA's documentation!
    :hidden:
 
    intro/index
+   intro/tutorial_page
```

Comment:
Since we are planning on adding multiple tutorials, I would actually take this opportunity to create the `tutorials` directory and rename this page to `tutorials/index.rst`. Then move the current `intro/tutorial.md` to `tutorials/basic.md`.

Reply:
Agreed, makes sense..

Reply:
Thank you @sphuber for your good suggestion. I applied this change. 
However, tutorial without toctree feels very weird, especially now that it has its own directory. 
please check now with the new commit, and let me know if you agree.